src/Views/FirmwareView.vala

```diff
@@ -180,6 +197,25 @@ public class About.FirmwareView : Gtk.Stack {
         update_list.show_all ();
     }
 
+    private void on_update (Fwupd.Device device, Fwupd.Release release) {
+        update_start (device);
+
+        update.begin (device, release, (obj, res) => {
+            update.end (res);
+            update_end ();
```

Comment:
This doesn't appear to be used anywhere outside of this function. Probably not necessary to create a function for two lines right?